src/utils/AnimationUtils.js

```diff
@@ -58,11 +60,11 @@ define(function (require, exports, module) {
         function cleanup() {
             $target
                 .removeClass(animClass)
-                .off("webkitTransitionEnd", finish);
+                .off("webkitTransitionEnd transitionEnd", finish);
```

Comment:
Important @le717: `transitionend` has to be all-lowercase (Same goes for below).